snapshots/cardano-1.21.1.yaml

```diff
@@ -0,0 +1,148 @@
+name: cardano-1.21.1
+
+resolver: lts-14.25
```

Comment:
Are we not going to use GHC 8.10.2 now?

Reply:
Not yet for 1.21.*. The upgrade is planned for the current sprint, but I believe we might have to wait for GHC 8.10.3 because of [Windows issues](https://twitter.com/ndm_haskell/status/1313181853822717952).